gubernator/templates/build_list.html

```diff
@@ -4,21 +4,27 @@
 <h1><a href="/jobs{{job_dir|dirname|dirname}}">{{job_dir|dirname|dirname}}</a>/{{job}}</h1>
 % endblock
 % block content
+<div id="content">
+% if testgrid_query
+<a href="{{testgrid_query | tg_url}}"><h3>Testgrid History for Job</h3></a>
```

Comment:
That should be tested automatically Real Soon Now, so I'm not going to worry about putting the functionality here.